### PR TITLE
fix: clean up table styling

### PR DESCRIPTION
**Pull request by patrickhulce**
R: @paulirish all

**Before**
image

**After**
image

### CODE REVIEW

File: lighthouse-core/report/report-generator.js [hunk 1]
```diff
@@ -111,6 +111,20 @@ class ReportGenerator {
       }
     });
 
+    Handlebars.registerHelper('kebabCase', function(str) {
```
Comment (patrickhulce) [inline lighthouse-core/report/report-generator.js]: would love to rebase against  #1660 and add tests here

File: lighthouse-core/formatters/partials/table.html [hunk 2]
```diff
@@ -7,15 +7,15 @@
...
+          <th class="table-column table-column-{{ kebabCase @key }}">{{this}}</th>
         {{/each}}
       </tr>
     </thead>
     <tbody>
       {{#each rows}}
         <tr>
           {{#each cols}}
-            <td>{{ sanitize this}}</td>
+            <td class="table-column table-column-{{ kebabCase (lookup ../../headingKeys @index) }}">{{ sanitize this}}</td>
```
Comment (patrickhulce) [inline lighthouse-core/formatters/partials/table.html]: don't like this, but not sure it's worth restructuring the format of rows just for class names here

File: lighthouse-core/formatters/partials/table.css [hunk 3]
```diff
@@ -43,11 +38,23 @@
 .table_list em + code, .table_list em + pre {
   margin-top: 10px;
 }
-.table_list td img {
+.table_list .table-column {
+  text-align: right;
+}
+.table_list .table-column.table-column-url {
+  text-align: left;
+  width: 250px;
```
Comment (patrickhulce) [inline lighthouse-core/formatters/partials/table.css]: could add more here, but on smaller screen sizes the optimized images table starts getting pretty squished, min-width doesn't seem to have an impact on fixed table width

File: lighthouse-core/formatters/partials/table.html (hunk 2 above)
Comment (ebidel) [inline lighthouse-core/formatters/partials/table.html]: Nice. It's useful to be able to target these. It's hard with a generic table, but specific data that might nee to look different here/there..

**Comment by ebidel** [comment]
This LGTM. One thing I notice is the `text-align: right` makes the code snippets funky:

screen shot 2017-02-15 at 9 46 08 am

vs before:

screen shot 2017-02-15 at 9 46 12 am

**Comment by patrickhulce** [comment]
oh thanks good catch @ebidel, fixed

**Comment by ebidel** [comment]
LGTM

**Pull request merged**